### DISCUSSION TITLE
How many CPU cores are in the Security Onion hardware?

### DISCUSSION THREAD

**Discussion by Ryan8bit** [Security-Onion-Solutions/securityonion#10346] [General]
I was wondering how many CPU cores are in each of the Security Onion hardware machines. Thanks!

**Comment by dougburks** [comment]
If you are interested in purchasing SOS hardware, please reach out to our sales team using the contact form at:
https://securityonionsolutions.com/hardware/

Otherwise, if you are using your own hardware, please reference the hardware requirements at:
https://docs.securityonion.net/en/2.3/hardware.html
